html/janus.js

```diff
@@ -2384,17 +2381,19 @@ function Janus(gatewayCallbacks) {
 							});
 						return;
 					}
+					// FIXME what is the correct scope of
+					// callbackUserMedia and getScreenMedia?
```

Comment:
the scope is the one where the arguments of `streamsDone` are visibile.

Reply:
I shall strip this out. Nesting a function below the top level of another function fires this warning, https://eslint.org/docs/rules/no-inner-declarations